Training6/thongnq-training6-task2/src/state/reducers/tasks.js

```diff
@@ -0,0 +1,69 @@
+import {
+    CREATE_TASK,
+    UPDATE_STATUS_SUBMITTING,
+    UPDATE_STATUS_READY,
+    UPDATE_STATUS_ERROR,
+    UPDATE_STATUS_SUCCESS,
+    UPDATE_STATUS_ERROR_SUBMIT
+} from "../actions/types";
+
+import {v4 as uuidv4} from 'uuid';
+
+const initialState = [];
+
+const tasks = (state = initialState, action) => {
+    const {type, payload} = action;
+
+    switch (type) {
+        case CREATE_TASK:
+            return [...state, {name: payload.name,
+                            status: "Draft",
+                            id: uuidv4()}];
+
+        case UPDATE_STATUS_READY:
```

Comment:
@Thong-Nguyen-Zigvy  you should declare only one action for update status, it look at the same, you can pass status into action payload